kibana-reports/public/application.tsx

```diff
@@ -20,16 +20,15 @@ import { AppPluginStartDependencies } from './types';
 import { OpendistroKibanaReportsApp } from './components/app';
 
 export const renderApp = (
-  { notifications, http, chrome }: CoreStart,
-  { navigation }: AppPluginStartDependencies,
+  { http, chrome, uiSettings }: CoreStart,
+  {  }: AppPluginStartDependencies,
```

Comment:
Do we need to `data` here?

Reply:
No, the parameter for `AppPluginStartDependencies` was `navigation` previously, which is not used